.github/CONTRIBUTING.rst

```diff
@@ -134,15 +143,23 @@ This is setting the CMake variable ``STATIC_LINKING`` to value ``True`` so CMake
 library, and is passing ``-j8`` flag to the underlaying build system, which in this case is Makefile, telling it that we want to
 build in parallel, using 8 processes.
 
+*N.B. on MacOS:*, you may need to turn off static linking and specify your platform name, e.g.:
+
+.. code::
+
+   python ./setup.py bdist_wheel --plat-name macosx-10.9-x86_64 -- -DSTATIC_LINKING=False -- -j8
```

Comment:
This is actually the default, so there's no need for specifying STATIC_LINKING=False

Reply:
ok i removed that.